### PR TITLE
feat: add support for Kubernetes 1.12.7

### PR DESCRIPTION
**Pull request by mboersma**
**Reason for Change**:

See https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.12.md#v1127

**Issue Fixed**:

**Requirements**:

- [x] Windows artifacts uploaded to acsmirror blob store
- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: pkg/api/common/versions.go [hunk 1]
```diff
@@ -98,8 +98,9 @@ var AllKubernetesSupportedVersions = map[string]bool{
 	"1.12.1":         false,
 	"1.12.2":         false,
 	"1.12.4":         false,
-	"1.12.5":         true,
+	"1.12.5":         false,
 	"1.12.6":         true,
+	"1.12.7":         true,
```
Comment (CecileRobertMichon) [inline pkg/api/common/versions.go]: I think you forgot to add it to the vhd script

Comment (jackfrancis) [reply pkg/api/common/versions.go]: Done @mboersma FYI

**Pull request merged**